src/pudl/transform/eia861.py

```diff
@@ -1258,6 +1273,138 @@ def demand_side_management(tfr_dfs):
         dict: A dictionary of transformed EIA 861 dataframes, keyed by table name.
 
     """
```

Comment:
can you add a little bit of explanation to the docs here. this table is weird so it'll be good to get down a little bit of your familiarity with it here.